inference-engine/ie_bridges/java/README.md

```diff
@@ -11,7 +11,7 @@ sudo apt-get install -y openjdk-11-jdk
 
 * Ubuntu 16.04
 ```bash
-sudo apt-get install -y openjdk-9-jdk
+sudo apt-get install -y openjdk-8-jdk
```

Comment:
Is it ok, that you changed the Java version?

Reply:
I've tested with such version, while 9 does not work for me

Reply:
I think, we can use just ```sudo apt-get install default-jdk``` for all ubuntu versions.

Reply:
agree, fixed.